libs/tree-sitter-wing/grammar.js

```diff
@@ -280,9 +280,20 @@ module.exports = grammar({
         "=",
         field("value", $.expression),
         field("block", $.block),
+        repeat(field("elif_let_block", $.elif_let_block)),
         optional(seq("else", field("else_block", $.block)))
       ),
 
+    elif_let_block: ($) =>
+      seq(
+        "elif let",
```

Comment:
nit: I'm not 100% sure, but I think tree-sitter handles things like this better when each has its own token

```suggestion
        "elif",
        "let",
```

Reply:
Thank you for pointing that out, @MarkMcCulloh. You're right, if I put 2 spaces between the `elif` and the `let`, an error occurs if I use the string ("elif let"). However, separating the tree-sitter takes care of this type of detail.

It's becoming a habit for you to correct me, keep it up! 😁